tsl/test/expected/data_node.out

```diff
@@ -1273,11 +1273,22 @@ BEGIN
 END
 $BODY$;
 CREATE EXTENSION timescaledb VERSION '0.0.0';
-\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+\c :TEST_DBNAME :ROLE_SUPERUSER;
 \set ON_ERROR_STOP 0
 SELECT * FROM add_data_node('data_node_1', 'localhost', database => 'data_node_1',
                             bootstrap => false);
 ERROR:  remote PostgreSQL instance has an incompatible timescaledb extension version
+-- Using oneself as a data node would create a cycle if allowed.
```

Comment:
I am confused by this comment. Do you mean that creating the cycle can be allowed?

Reply:
No, that is what this fix prevents by giving an error instead of attempting to create a self-reference.

Reply:
Can you improve this comment then? E.g.:
```suggestion
-- Test preventing from using oneself as a data node to avoid a cycle.
```

Reply:
Improved.